ouroboros-consensus-test/test-consensus/Test/Consensus/HardFork/Combinator/B.hs

```diff
@@ -418,6 +432,7 @@ instance Serialise (SerialisedHeader BlockB) where
   SerialiseNodeToClient
 -------------------------------------------------------------------------------}
 
+instance SerialiseNodeToClient BlockB ()
```

Comment:
~~Why is this needed? Serialization for `()` is suspect; it's usually something we'll just omit from the serialization instead of using `TkNull`. And that means there's a `toCBOR` or `fromCBOR` that should be counted against the list length, etc etc.~~

Ah, it's because `type instance LedgerCfg (LedgerState BlockB) = ()` -- no surprise. This is probably fine for such a testing module. But it'd be "safer" to declare `data LedgerCfgB = LedgerCfgB` and only provide instances for that :/